html/entities.ts

```diff
@@ -93,7 +93,7 @@ export function unescape(
   }
 
   return str
-    .replaceAll(entityRe, (m) => entityList[m])
+    .replaceAll(entityRe, (m) => entityList[m]!)
```

Comment:
Can you please justify why `entityList[m]` is non-null?

Reply:
`entityList` is filled with the `defaultEntityList` and the values in `entityRe` are generated from the keys of `entityList`, so I think `entityList[m]` with `m` an element of `entityRe` is always non-null